packages/ses/index.d.ts

```diff
@@ -67,6 +67,7 @@ export type FinalStaticModuleType =
 export interface RedirectStaticModuleInterface {
   record: FinalStaticModuleType;
   specifier: string;
+  importMeta?: any;
 }
 
 export type StaticModuleType =
```

Comment:
This part will need a rewrite to match current spec. That's going to happen alongside a wider rewrite to match shapes defined in the compartment proposal later.